int32 tag
float32 score
geometry_msgs/Point[] bbox_corners